frontend/src/container/Dashboard.jsx

```diff
@@ -403,23 +403,26 @@ class Dashboard extends React.Component {
 
                   {idRoleUserWorkspace >= 2 && (
                     <div className='dashboard__calltoaction'>
-                      {props.appList.map(app => {
-                        const contentType = props.contentType.find(ct => app.slug.includes(ct.slug)) || {creationLabel: '', slug: ''}
-                        return (
-                          <ContentTypeBtn
-                            customClass='dashboard__calltoaction__button'
-                            hexcolor={app.hexcolor}
-                            label={app.label}
-                            faIcon={app.faIcon}
-                            // @fixme Côme - 2018/09/12 - trad key bellow is a little hacky. The creation label comes from api but since there is no translation in backend
-                            // every files has a 'externalTradList' array just to generate the translation key in the json files through i18n.scanner
-                            creationLabel={props.t(contentType.creationLabel)}
-                            onClickBtn={() => props.history.push(`${PAGE.WORKSPACE.NEW(props.curWs.id, contentType.slug)}?parent_id=null`)}
-                            appSlug={app.slug}
-                            key={app.label}
-                          />
-                        )
-                      })}
+                      {props.appList
+                        .filter(app => idRoleUserWorkspace === 2 ? app.slug !== 'contents/folder' : true)
```

Comment:
`idRoleUserWorkspace === 2` Does we should not use constants to store user workspace roles ? An issue can be opened for that refactor (it is not subject of this MR)

Reply:
yes, we should

Reply:
Can you open an issue ?

Reply:
https://github.com/tracim/tracim/issues/1396